app/javascript/components/mentoring/session/Guidance.tsx

```diff
@@ -108,8 +111,9 @@ export const Guidance = ({
               </p>
               <pre className="overflow-auto">
                 <code
+                  className={language}
                   dangerouslySetInnerHTML={{ __html: exemplarSolution }}
```

Comment:
Is `exemplarSolution` here already properly HTML escaped?

Reply:
Probably not

Reply:
@joshgoebel Does it need to be on a `<code>` elem though?

Reply:
Yes, always, HTML inside code is still HTML:

```
<code>
<script>
destroyWebsite()
</script>
</code>
```

Not good.

Reply:
https://github.com/exercism/website/pull/1886